ew/utils/loop.py

```diff
@@ -858,6 +859,87 @@ async def enemy_action_tick_loop(id_server):
         except Exception as e:
             ewutils.logMsg('Enemy tick loop failed. Reason:{}'.format(e))
 
+severityLimit = {
+    "low": 5,
+    "medium" : 15,
+    "high" : 25,
+    "wasted" : 50,
+    "wrecked" : 90,
+    "totalled" : 100
+}
+async def drunk_effects(id_server):
+    while not ewutils.TERMINATE:
+        timer = 30
+        await asyncio.sleep(timer)
+        users = bknd_core.execute_sql_query("SELECT inebriation, id_user FROM users WHERE inebriation >0")
+        server = ewcfg.server_list[id_server]
+        if users:
+            for inebriation, user in users:
+                user_data = EwUser(id_server=id_server, id_user=user)
+                player = EwPlayer(id_user=user)
+                inebriation=user_data.inebriation
+                mutations = user_data.get_mutations()
+                channel = fe_utils.get_channel(server,user_data.poi)
+                
+                chance = max(1, 50-inebriation)
+                #More frequently, the more drunk
+                if random.randint(1,chance)==1:
+                    user_data = EwUser(id_server=id_server, id_user=user)
+                    injury_text = ""
+                    slimeloss=0
+                    slime = user_data.slimes
+                    
+                    savingthrow = random.randint(1,20)
+                    # Lucky rolls with advantage
+                    if ewcfg.mutation_id_lucky in mutations:
+                        savingthrow = max(random.randint(1,20),random.randint(1,20)) 
```

Comment:
You've already called to random once so it'd make more sense to have this be "max(savingthrow, random.randinit(1,20))".